src/schema/type/DefinedTerm.graphql

```diff
@@ -69,4 +69,11 @@ type DefinedTerm implements ThingInterface {
     # TODO: This should only be a relation to one DefinedTermSet
     "https://pending.schema.org/inDefinedTermSet"
     inDefinedTermSet: [DefinedTermSet] @relation(name: "HAS_DEFINED_TERM", direction: IN)
+
+    #######################
+    ### SKOS properties ###
+    # When we use a DefinedTerm to refer to an annotation motivation, we have to add the movitavtion
+    # of which this term is a more specific version of to skos:broader. However,
```

Comment:
```suggestion
    # of which this term is a more specific version of to skos:broader.
```

Reply:
Ah, I'm not sure what I wanted to say here...